lib/ecto/multi.ex

```diff
@@ -132,8 +133,8 @@ defmodule Ecto.Multi do
 
   ## Example
 
-      iex> lhs = Ecto.Multi.new |> Ecto.Multi.run(:left, &{:ok, &1})
-      iex> rhs = Ecto.Multi.new |> Ecto.Multi.run(:right, &{:error, &1})
+      iex> lhs = Ecto.Multi.new |> Ecto.Multi.run(:left, fn _, _ -> {:ok, :ok} end)
```

Comment:
Separate from this PR, I've had a few instances where using the capture operator would be useful, but the restriction that it _must_ name all functions (can't use `&2` if `&1` isn't defined) is limiting.

Is there a restriction I'm unaware of as to why we don't allow just `&2` to be used, and silently discard `&1`?

Reply:
This is not the place for this discussion. :) You can start something in the mailing list but short answer is yes.